app/views/admin/users/_member_index.html.erb

```diff
@@ -21,7 +21,27 @@
       <tbody class="crayons-card">
         <% @users.each do |user| %>
         <tr>
-          <td><%= link_to user.name, admin_user_path(user) %></td>
+          <td>
+          <div class="flex s:items-start flex-col s:flex-row">
+            <a href="<%= admin_user_path(user.id) %>">
+              <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+            </a>
+            <%= user.name %>
+          </div>
+          <p class="fs-xs color-base-60">
+            @<%= user.username %> <%= user.current_roles %>
```

Comment:
I'm running into a bit of difficulty getting this particular column to be laid out like it is in the design... it's close, but not quite there yet!

Reply:
Let me know if I can help!

Reply:
I was finally able to get it a bit closer, but I don't think it's 100% there, @aitchiss! I would love your input on this.

Reply:
I'll take a look!